integration_test/gce/gce_testing.go

```diff
@@ -1173,6 +1181,10 @@ func IsRHEL(platform string) bool {
 	return strings.HasPrefix(platform, "rhel-")
 }
 
+func isRHEL7SAPHA(platform string) bool {
+	return platform == "rhel-7-7-sap-ha" || platform == "rhel-7-9-sap-ha"
```

Comment:
we should do something like "if it starts with 'rhel-' and ends with '-sap-ha'" instead of hardcoding 7-7 and 7-9.

Reply:
Ok! That makes more sense. Will update.

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1062/commits/62a78b43dd19318c2ffa2b884786833b72098a55 .

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1062/commits/774e41cf475c450b283d40dea91401d06d2c5fdb .